setup/index.php

```diff
@@ -14,92 +14,15 @@
-    }
-}
-if (!$isCommandLine) {
-    $https = isset($_SERVER['HTTPS']) ? $_SERVER['HTTPS'] : false;
-    $installBaseUrl= (!$https || strtolower($https) != 'on') ? 'http://' : 'https://';
-    $installBaseUrl .= $_SERVER['HTTP_HOST'];
-    if (isset($_SERVER['SERVER_PORT']) && (string)$_SERVER['SERVER_PORT'] != '' && $_SERVER['SERVER_PORT'] != 80) $installBaseUrl= str_replace(':' . $_SERVER['SERVER_PORT'], '', $installBaseUrl);
-    $installBaseUrl .= ($_SERVER['SERVER_PORT'] == 80 || ($https !== false || strtolower($https) == 'on')) ? '' : ':' . $_SERVER['SERVER_PORT'];
-    $installBaseUrl .= $_SERVER['SCRIPT_NAME'];
-    $installBaseUrl = htmlspecialchars($installBaseUrl, ENT_QUOTES, 'utf-8');
-    define('MODX_SETUP_URL', $installBaseUrl);
-} else {
-    define('MODX_SETUP_URL','/');
-}
-
-/* session loop-back tester */
-if (!$isCommandLine && (!isset($_GET['s']) || $_GET['s'] != 'set') && !isset($_SESSION['session_test'])) {
-    $_SESSION['session_test']= 1;
-    echo "<html><head><title>Loading...</title><script>window.location.href='" . MODX_SETUP_URL . "?s=set';</script></head><body></body></html>";
-    exit ();
-} elseif (!$isCommandLine && isset($_GET['s']) && $_GET['s'] == 'set' && !isset($_SESSION['session_test'])) {
-    die('<html><head><title></title></head><body><h1>FATAL ERROR: MODX Setup cannot continue.</h1><p>Make sure your PHP session configuration is valid and working.</p></body></html>');
-}
-
-$setupPath= strtr(realpath(dirname(__FILE__)), '\\', '/') . '/';
-define('MODX_SETUP_PATH', $setupPath);
-$installPath= strtr(realpath(dirname(__DIR__)), '\\', '/') . '/';
-define('MODX_INSTALL_PATH', $installPath);
-
-if (!include(MODX_SETUP_PATH . 'includes/config.core.php')) {
-    die('<html><head><title></title></head><body><h1>FATAL ERROR: MODX Setup cannot continue.</h1><p>Make sure you have uploaded all of the setup/ files; your setup/includes/config.core.php file is missing.</p></body></html>');
-}
-if (!include(MODX_SETUP_PATH . 'includes/modinstall.class.php')) {
-    die('<html><head><title></title></head><body><h1>FATAL ERROR: MODX Setup cannot continue.</h1><p>Make sure you have uploaded all of the setup/ files; your setup/includes/modinstall.class.php file is missing.</p></body></html>');
-}
 $modInstall = new modInstall();
 if ($modInstall->getService('lexicon','modInstallLexicon')) {
     $modInstall->lexicon->load('default');
 }
 //$modInstall->findCore();
 $modInstall->doPreloadChecks();
-$requestClass = $isCommandLine ? 'request.modInstallCLIRequest' : 'request.modInstallRequest';
+$requestClass = MODX_SETUP_INTERFACE_IS_CLI ? 'request.modInstallCLIRequest' : 'request.modInstallRequest';
```

Comment:
Where is MODX_SETUP_INTERFACE_IS_CLI defined? I can't find this change in these diffs.

Reply:
> Where is MODX_SETUP_INTERFACE_IS_CLI defined? I can't find this change in these diffs.

`setup/provisioner/bootstrap.php` line 11:
https://github.com/modxcms/revolution/pull/14999/files#diff-6d29bb4ac4a6b37d567e2441cf473fe0R11